toolkits/global/packages/global-layout-cluster/scss/50-components/layout-cluster.scss

```diff
@@ -0,0 +1,36 @@
+.l-cluster {
+	display: flex;
+	@supports (gap: 1em) {
+		gap: var(--cluster--gap, #{$cluster--gap});
+	}
+}
+
+.l-cluster > * {
+	/* ↓ Because margin doubles up */
+	margin: calc(#{$cluster--gap} / 2);
+	@supports (gap: 1em) {
+		margin: 0;
+		gap: var(--cluster--gap, #{$cluster--gap});
+	}
+}
+
+.l-cluster--right {
```

Comment:
I know `justify-content` doesn't make use of logical properties. But I wonder if moving toward that naming convention would be a good idea.

So, for `l-cluster--right` there would be `l-cluster--inline-end`.